<launch>
<param name="use_sim_time" value="true"/>
<node pkg="slam_diy" type="main" name="slam_gmapping" output="screen">
    <param name="transform_publish_period" value = "0.05"/>
    <param name="base_frame" value="base_link"/>
    <param name="map_frame" value="map"/>
    <param name="odom_frame" value="odom"/>
</node>
<!--node pkg="tf" type="static_transform_publisher" name="base_link_to_laser" args=" 0 0 0 0 0 0 base_link base_laser 100" /-->
</launch>
